### PR TITLE
Fixed few more UI test failures

### PR DESCRIPTION
**Pull request by sghai**
- fixed `test_remove_template` by associating org with template
- fixed `test_create_subnet_3` by assciating org with domain

Issue is that when we create any entity without associating an org then it won't be visible globally if any org is selected. For ex: If you create a template in 'Any context' then that template won't be visible under any selected org. 

So I fixed the tests by associating the orgs with all entities.

### CODE REVIEW

**Comment by sghai** [comment]
```
@Test: Remove a template ... ok

----------------------------------------------------------------------
Ran 1 test in 72.638s

OK
```

**Comment by sghai** [comment]
```
@Test: Create new subnet and associate domain with it ... ok

----------------------------------------------------------------------
Ran 1 test in 85.306s

OK
```

**Comment by jyejare** [comment]
ACK.

**Comment by sghai** [comment]
```
@Test: Update OS medium ... ok

----------------------------------------------------------------------
Ran 1 test in 82.026s

OK
```

File: tests/foreman/ui/test_operatingsys.py [hunk 1]
```diff
@@ -20,6 +20,13 @@
 class OperatingSys(UITestCase):
     """Implements Operating system tests from UI"""
 
+    @classmethod
+    def setUpClass(cls):  # noqa
+        org_attrs = entities.Organization().create()
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_operatingsys.py]: `s/create/create_json/`

File: tests/foreman/ui/test_subnet.py [hunk 2]
```diff
@@ -18,6 +18,13 @@
 class Subnet(UITestCase):
     """Implements Subnet tests in UI"""
 
+    @classmethod
+    def setUpClass(cls):  # noqa
+        org_attrs = entities.Organization().create()
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_subnet.py]: `s/create/create_json/`

File: tests/foreman/ui/test_template.py [hunk 3]
```diff
@@ -17,6 +17,13 @@
 class Template(UITestCase):
     """Implements Provisioning Template tests from UI"""
 
+    @classmethod
+    def setUpClass(cls):  # noqa
+        org_attrs = entities.Organization().create()
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_template.py]: `s/create/create_json/`

File: tests/foreman/ui/test_subnet.py [hunk 4]
```diff
@@ -76,10 +83,13 @@ def test_create_subnet_3(self):
         name = gen_string("alpha", 4)
         network = gen_ipaddr(ip3=True)
         mask = gen_netmask()
-        domain_name = entities.Domain().create()['name']
+        domain_name = entities.Domain(
+            organization=[self.org_id]
+        ).create()['name']
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_subnet.py]: `s/create/create_json/`

**Comment by Ichimonji10** [comment]
ACK pending comments.

Can you rebase too? A rebase doesn't need to happen right *now*, but by the time this PR is updated when you get in to work tomorrow, I expect a few more merges will have been made.

**Comment by sghai** [comment]
Sure..thanks.. Comments are addressed..please review again..

**Comment by Ichimonji10** [comment]
Looks great. Thank you. ACK

**Pull request merged**